packages/geoprocessing/src/toolbox/geoblaze/rasterStats.ts

```diff
@@ -8,6 +8,7 @@ import { Feature, MultiPolygon, FeatureCollection } from "@turf/helpers";
 import geoblaze, { Georaster } from "geoblaze";
 import { StatsObject, CalcStatsOptions } from "../../types/geoblaze";
 import { toRasterProjection, defaultStatValues } from "./geoblaze";
+import { forEach } from "lodash";
```

Comment:
Lodash is a big import.  Typically you will want to import just the function you need.  For example `import forEach from "lodash/forEach"`.  Is there a reason to use it in this case instead of built-in functions?  Array.forEach or Object.keys(object).forEach().